css/portal-dashboard/feedback/feedback-rows.less

```diff
@@ -77,8 +77,8 @@
         min-height: 50px;
         outline: none;
         padding: 5px 10px 13px;
-        resize: none;
-        width: calc(100% - 20px);
+        resize: vertical;
```

Comment:
`resize` should be set to `none` as of this PR https://github.com/concord-consortium/portal-report/pull/298.